docs/notebooks/bayesian_t-test.ipynb

```diff
@@ -38,14 +38,17 @@
    "cell_type": "markdown",
```

Comment:
<p>add sentence about how much different are the means, not just that they are different</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>